### ISSUE TITLE
[BUG] - Wrong texture is displayed in the texture modal (gallery)

### ISSUE THREAD

**Issue by Juknum** [Faithful-Resource-Pack/App#85]
**Describe the bug**
When opening a texture modal inside the gallery, the shown images are the ones from the latest branch (1.19 actually), instead, it should be the textures from the selected branch

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to https://webapp.faithfulpack.net/#/gallery/java/32x/1.18.2/all?show=1
2. Right Click on any image on the left -> open the image in a new tab
3. See the URL being : https://raw.githubusercontent.com/CompliBot/Default-Java/1.19/assets/realms/textures/gui/realms/configure_icon.png 
4. Instead of https://raw.githubusercontent.com/CompliBot/Default-Java/1.18.2/assets/realms/textures/gui/realms/configure_icon.png

**Expected behaviour**
The displayed images should correspond to the selected version

**Screenshots**
N/A

**Browser & Environment (computer/phone):**
 N/A

**Logs**
N/A

**Additional context**
N/A

**Comment by Pomi108** [comment]
Oh nice, I'd wanted to report this one myself but forgot to do it. Ty for doing this x)

**Comment by Pomi108** [comment]
Try it with any version below 1.14, so that programmer art is the default. When you click on it, the jappa texture will be shown

**Comment by RobertRR11** [comment]
yes just found out, I had actually tested it first with the ore textures, but those are separate entries in the database

**Comment by TheRolfFR** [comment]
We gonna hit 2 birds with 1 stone making use of a new API endpoint at the same time

**Comment by TheRolfFR** [comment]
With 177c603, but now I have to display them all, so it renders like this:
![image](https://user-images.githubusercontent.com/18086786/181723766-2c0dd67f-3f01-4ac7-bcc8-8d392fbab3f0.png)
And on mobile, it gives that:
![image](https://user-images.githubusercontent.com/18086786/181723978-03066971-5380-4a78-9cd8-b610563800cd.png)
What do you think ?

**Comment by Pomi108** [comment]
This display is cool, but please don't call it "progart". Use the proper "programmer art" instead

Also, does this change fix the latest texture displaying instead of the one from the specific version?

**Comment by RobertRR11** [comment]
> Also, does this change fix the latest texture displaying instead of the one from the specific version?

yes
